### PR TITLE
Beta 13 Release

### PR DESCRIPTION
**Pull request by fubuloubu**
Please add suggestions to what we should mention in the release notes for b13.

### CODE REVIEW

File: docs/release-notes.rst [hunk 1]
```diff
@@ -1,6 +1,20 @@
 .. _release-notes:
 
 Release Notes
+#############
+v0.1.0-beta.13
+**************
+
+Date released: 25-09-2019
```
Comment (fubuloubu) [inline docs/release-notes.rst]: Release tomorrow?

**Comment by iamdefinitelyahuman** [comment]
* VIP: vyper-json https://github.com/ethereum/vyper/issues/1520 / https://github.com/ethereum/vyper/pull/1594
* VIP: environment variables and constants as default parameters https://github.com/ethereum/vyper/issues/1525 / https://github.com/ethereum/vyper/pull/1607
* type checks for default params and arrays https://github.com/ethereum/vyper/pull/1596
* bugfix: assertions in side for loops https://github.com/ethereum/vyper/pull/1619

That's it from me.  Not sure if they're all worthy of changelog mention.

**Comment by charles-cooper** [comment]
- Fix zero padding for ABI encoder https://github.com/ethereum/vyper/pull/1611
- Check calldatasize before calldataload for function selector https://github.com/ethereum/vyper/pull/1606

**Comment by iamdefinitelyahuman** [comment]
#1597 should be included in this release as well, yeah?

**Comment by charles-cooper** [comment]
> #1597 should be included in this release as well, yeah?

If it gets reviewed .. ;)

**Comment by fubuloubu** [comment]
One more review and I'll merge it ;)

**Comment by fubuloubu** [comment]
> #1597 should be included in this release as well, yeah?

Added #1493

**Comment by fubuloubu** [comment]
@charles-cooper could you clean up #1493 with some of the decisions that were made in that conversation? I'm not sure if it was meant as a VIP or not because the template wasn't really ever filled out.

**Comment by charles-cooper** [comment]
> @charles-cooper could you clean up #1493 with some of the decisions that were made in that conversation? I'm not sure if it was meant as a VIP or not because the template wasn't really ever filled out.

@fubuloubu filled it out a bit. Let me know if you want to see anything else.

**Comment by fubuloubu** [comment]
@charles-cooper looks good. Sometimes I wish we got rid of either the summary of the abstract, kinda duplicate.

File: docs/release-notes.rst (hunk 1 above)
Comment (fubuloubu) [reply docs/release-notes.rst]:
```suggestion
Date released: 27-09-2019
```

**Comment by jacqueswww** [comment]
Wow that release label is really green, looks good to me.

**Pull request merged**